packages/editor/src/components/global-styles-provider/index.js

```diff
@@ -55,14 +55,30 @@ function useGlobalStylesUserConfig() {
 			const _globalStylesId =
 				select( coreStore ).__experimentalGetCurrentGlobalStylesId();
 
+			// We want the global styles ID request to finish before triggering
+			// the OPTIONS request for user capabilities, otherwise it will
+			// fetch `/wp/v2/global-styles` instead of
+			// `/wp/v2/global-styles/{id}`!
+			if ( ! _globalStylesId ) {
+				return { isReady: false };
```

Comment:
So what I think is going on is that `isReady` is being used as a "has resolved". So it if never resolves, the [global styles provider will return null](https://github.com/WordPress/gutenberg/blob/2da07af0d9d7e8dcb6c1fac2388e7134e498be66/packages/editor/src/components/global-styles-provider/index.js#L232-L232). 

If I flip it to `true` then hybrid themes will load the pattern editor.

🤔 🤔 

Hybrid themes don't have a user config. All I can think of is a `getEditorSettings().__unstableIsBlockBasedTheme` check or maybe keep the `userCanEditGlobalStyles === 'boolean'` check but set `userCanEditGlobalStyles` to `null` if `_globalStylesId` doesn't exist?

```js
const userCanEditGlobalStyles = _globalStylesId
				? canUser( 'update', {
						kind: 'root',
						name: 'globalStyles',
						id: _globalStylesId,
				  } )
				: null;
```

Whaddya reckon?

Reply:
That sounds great!